server/routes/statistics.js

```diff
@@ -242,36 +237,16 @@ module.exports = function (config) {
 		}),
 		'get-flow-rejections-data': function (unresolvedQuery) {
 			return rejectionsQueryHandler.resolve(unresolvedQuery)(function (query) {
-				return getData(driver)(function (data) {
-					var fullSize;
-
-					data = sortData(
-						filterBusinessProcesses(data.businessProcesses, assign({
-							flowStatus: 'rejected'
-						}, query)),
-						function (bpA, bpB) {
-							return bpA.rejectedDateTime - bpB.rejectedDateTime;
-						}
-					);
-
-					if (!data.length) {
-						return { size: 0, view: [] };
-					}
-
-					fullSize = data.length;
-
-					data = getPage(data, query.page);
-
-					return getViewRecords(
-						data,
-						rejectionsListProperties,
-						rejectionsListComputedProperties
-					)(function (result) {
-						result.size = fullSize;
-
-						return result;
-					});
-				});
+				var data = { rows: [], pageCount: 1 };
+				data.pageCount = getPage(data.rows, query.page);
+				/* TODO inflate data.rows with real results
+					expected structure:
+					[
+					['Reason 1, Reason 2', '*', 3, 'John Smith', 'Revision', '02/03/2017', 'Super inc.'],
```

Comment:
@mtuchowski that was exactly what I wanted to achieve: no manipulation of data on the view, jsut display. This way we force controller to give us all we need, and all the views that will follow (pdf, csv) will have an easier task with parsing. So I would rather leave it as is.